src/syft/core/node/common/service/obj_search_service.py

```diff
@@ -194,9 +195,18 @@ def process(
         results: List[Pointer] = list()
 
         for obj in node.store.get_objects_of_type(obj_type=object):
+
+            # if this tensor allows anyone to search for it, then one of its keys
+            # has an All() class in it.
+            contains_all_in_permissions = False
+            for key in obj.search_permissions.keys():
+                if isinstance(key, All):
```

Comment:
nit: I *think* `if any(isinstance(key, All) for key in obj.search_permissions)` is a bit clearer

Reply:
I'l make this adjustment!